certified-connectors/KyndrylMainframe/apiDefinition.swagger.json

```diff
@@ -0,0 +1,707 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "Kyndryl mainframe connector",
+    "version": "1.0.0",
+    "title": "kyndryl"
+  },
+  "host": "kyndryl-apigateway.azure-api.net",
+  "basePath": "/",
+  "schemes": [
+    "http"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/cb12-policy/getpolicy": {
+      "get": {
+        "tags": [
+          "cb12-policy"
+        ],
+        "operationId": "getPolicy",
+        "parameters": [
+          {
+            "name": "CUSTOMERNUMBER",
+            "in": "query",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/getPolicy_response_200"
+            }
+          }
+        },
+        "summary": "Get policy"
+      }
+    },
+    "/customernumber/Custdetailadd": {
+      "post": {
+        "tags": [
+          "customernumber"
+        ],
+        "operationId": "postCustomerdetailsupd",
+        "parameters": [
+          {
+            "in": "body",
+            "name": "postCustomerdetailsupd_request",
+            "description": "request body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/postCustomerdetailsupd_request"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/postCustomerdetailsupd_response_200"
+            }
+          }
+        },
+        "summary": "Post customer details"
+      }
+    },
+    "/customernumber/Custdetailupd/{num}/{firstname}": {
+      "put": {
+        "tags": [
+          "customernumber"
+        ],
+        "operationId": "putCustomerdetail",
+        "parameters": [
+          {
+            "name": "num",
```

Comment:
@RTkyndryl for all parameters The 'description' property is required and the 'x-ms-summary' property is required.
Please add both for each parameter in the swagger file.